bootstrap.min.css

```diff
@@ -1,11 +0,0 @@
-@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500,700");/*!
```

Comment:
Deleted, not needed anymore since it's included through bs_theme which is included through webpack/bootstrap loader